src/Components/MoodCard.js

```diff
@@ -2,24 +2,14 @@ import React, { useContext } from "react";
 import { UserContext } from "../Contexts/UserContext";
 import setImages from "../Utilities/SetImages";
 
-const MoodCard = () => {
```

Comment:
Refactored. Realized this logic shouldn't go here but instead should be in a container that holds the Cards.